sympy/integrals/laplace.py

```diff
@@ -1561,12 +1608,345 @@ def _inverse_laplace_time_diff(F, s, t, plane):
+        elif poles[0][n] == -2 and poles[0][m] == S.Half:
+            # 1/(a0*sqrt(s)+b0)**2 == 1/a0**2 * 1/(sqrt(s)+b0/a0)**2
+            a_sq = poles[0][b]/poles[0][a]
+            a_ = a_sq**2
+            k_ = 1/poles[0][a]**2*constants
+            if a_sq.is_positive:
+                result = (
+                    k_*(1 - 2/sqrt(pi)*sqrt(a_)*sqrt(t) +
+                        (1-2*a_*t)*exp(a_*t)*(erf(sqrt(a_)*sqrt(t))-1)))
+                debugf('[ILT _i_l_i] Rule (10) returns %s', (result, ))
+        elif poles[0][n] == -3 and poles[0][m] == S.Half:
+            # 1/(a0*sqrt(s)+b0)**3 == 1/a0**3 * 1/(sqrt(s)+b0/a0)**3
+            a_ = poles[0][b]/poles[0][a]
+            k_ = 1/poles[0][a]**3*constants
+            if a_.is_positive:
+                result = (
+                    k_*(2/sqrt(pi)*(a_**2*t+1)*sqrt(t) -
+                        a_*t*exp(a_**2*t)*(2*a_**2*t+3)*erfc(a_*sqrt(t))))
+                debugf('[ILT _i_l_i] Rule (13) returns %s', (result, ))
+        elif poles[0][n] == -4 and poles[0][m] == S.Half:
+            # 1/(a0*sqrt(s)+b0)**4 == 1/a0**4 * 1/(sqrt(s)+b0/a0)**4
+            a_ = poles[0][b]/poles[0][a]
+            k_ = 1/poles[0][a]**4*constants/3
+            if a_.is_positive:
+                result = (
+                    k_*(t*(4*a_**4*t**2+12*a_**2*t+3)*exp(a_**2*t) *
+                        erfc(a_*sqrt(t)) -
+                        2/sqrt(pi)*a_**3*t**(S(5)/2)*(2*a_**2*t+5)))
+                debugf('[ILT _i_l_i] Rule (16) returns %s', (result, ))
+        elif poles[0][n] == -S.Half and poles[0][m] == 2:
+            # 1/sqrt(a0*s**2+b0) == 1/sqrt(a0) * 1/sqrt(s**2+b0/a0)
+            a_ = sqrt(poles[0][b]/poles[0][a])
+            k_ = 1/sqrt(poles[0][a])*constants
+            result = (k_*(besselj(0, a_*t)))
+            debugf('[ILT _i_l_i] Rule (35), (44) returns %s', (result, ))
+
+    elif len(poles) == 1 and len(zeros) == 1:
+        if (
+                poles[0][n] == -3 and poles[0][m] == S.Half and
+                zeros[0][n] == S.Half and zeros[0][b] == 0):
+            # sqrt(az*s)/(ap*sqrt(s+bp)**3)
+            # == sqrt(az)/ap * sqrt(s)/(sqrt(s+bp)**3)
+            a_ = poles[0][b]
+            k_ = sqrt(zeros[0][a])/poles[0][a]*constants
+            result = (
+                k_*(2*a_**4*t**2+5*a_**2*t+1)*exp(a_**2*t) *
+                erfc(a_*sqrt(t)) - 2/sqrt(pi)*a_*(a_**2*t+2)*sqrt(t))
+            debugf('[ILT _i_l_i] Rule (14) returns %s', (result, ))
+        if (
+                poles[0][n] == -1 and poles[0][m] == 1 and
+                zeros[0][n] == S.Half and zeros[0][m] == 1):
+            # sqrt(az*s+bz)/(ap*s+bp)
+            # == sqrt(az)/ap * (sqrt(s+bz/az)/(s+bp/ap))
+            a_ = zeros[0][b]/zeros[0][a]
+            b_ = poles[0][b]/poles[0][a]
+            k_ = sqrt(zeros[0][a])/poles[0][a]*constants
+            result = (
+                k_*(exp(-a_*t)/sqrt(t)/sqrt(pi)+sqrt(a_-b_) *
+                    exp(-b_*t)*erf(sqrt(a_-b_)*sqrt(t))))
```

Comment:
Many of these formulae involve square roots and things. These can be problematic because branch cuts so there can be differences between `1/sqrt(a)` vs `sqrt(1/a)` or `sqrt(a)*sqrt(b)` vs `sqrt(a*b)` etc. Are these all supposed to hold for both negative, positive and non-real values of `a`, `b` etc?

I suppose that `t` is always nonnegative. Is it okay that this formula has a singularity at `t=0`?

Reply:
I will look at all occurrences of `sqrt` and check whether the branch cuts are OK.

Reply:
> I suppose that `t` is always nonnegative. Is it okay that this formula has a singularity at `t=0`?

Yes, due to the `Heaviside(t)` in the result :)))

Reply:
I have looked at all cases, and consider it OK

In the table below, `OK` means there is no `sqrt` branch involved.

`only determines k_` means that only the pre-factor of the result is influenced by the branch: it then can be the difference between $i\sqrt{x}$ and $\sqrt{-x}$, and generally happens if the term with $s$ has a negative sign.

`only gets positive arguments` means a check prevents the wrong branch being used.

`but manually tested` means I checked all combinations of signs with ILT and LT and verified the result.

So I think this is good to go.


```
(1)  sqrt branch only detemines k_
(2)  one sqrt only gets positive arguments, the other sqrt branch only detemines k_
(4)  OK
(5)  OK
(6)  OK
(7)  sqrt branch only detemines k_
(8)  OK
(9)  sqrt branch only detemines k_
(10) sqrt only gets positive arguments
(11) sqrt only gets positive arguments
(12) sqrt branch only detemines k_
(13) OK
(14) sqrt branch only detemines k_
(15) sqrt branch only detemines k_
(16) OK
(17) OK
(18) OK
(19) OK
(22) non-trivial, but manually tested for correct behaviour. sqrt branch influences k_
(23) non-trivial, but manually tested for correct behaviour. sqrt branch influences k_
(35, 44) The sign of the term in the sqrt implicitely decides whether the result contains $I_0$ or $J_0$. I think it is OK because meijerint would give the same result. sqrt branch influences k_
```